index.html

```diff
@@ -1,14 +1,74 @@
 <!DOCTYPE html>
 <html lang="en">
+
 <head>
-    <meta charset="UTF-8">
-    <meta http-equiv="X-UA-Compatible" content="IE=edge">
-    <meta name="viewport" content="width=device-width, initial-scale=1.0">
-    <title>LAMBDA</title>
+  <meta charset="UTF-8">
+  <meta http-equiv="X-UA-Compatible" content="IE=edge">
+  <meta name="viewport" content="width=device-width, initial-scale=1.0">
+  <link rel="preconnect" href="https://fonts.googleapis.com">
+  <link rel="preconnect" href="https://fonts.gstatic.com" crossorigin>
+  <link href="https://fonts.googleapis.com/css2?family=Yeseva+One&display=swap" rel="stylesheet">
+  <script type="module" src="https://unpkg.com/ionicons@5.4.0/dist/ionicons/ionicons.esm.js"></script>
+  <script nomodule="" src="https://unpkg.com/ionicons@5.4.0/dist/ionicons/ionicons.js"></script>
+  <link rel="stylesheet" href="css/style.css">
+  <title>LAMBDA - healthy food restaurant</title>
 </head>
+
 <body>
-    <h1>LAMBDA restaurant</h1>
-    
+  <div class="main-wrapper">
+    <header class="main-header">
+      <a class="logo" href="#">Lambda</a>
```

Comment:
```suggestion
      <a href="/">
        <h1 class="logo">Lambda</h1>
      </a>
```
See description why in another comment.
Link should pass us to the root I guess.